platform/javascript/os_javascript.h

```diff
@@ -83,6 +83,7 @@ class OS_JavaScript : public OS_Unix {
 	String get_executable_path() const;
 	virtual Error shell_open(String p_uri);
 	virtual String get_name() const;
+	virtual void add_frame_delay(bool p_can_draw) {}
```

Comment:
Might use a comment explaining that it's define as no-op to override the base `OS` implementation, which is redone in `main_loop_callback()`.

Reply:
Done